docs/en/roadmap/2020-12-roadmap.md

```diff
@@ -0,0 +1,27 @@
+# Roadmap
+
+Date: 2020-10-01 to 2021-12-30
```

Comment:
```suggestion
Date: 2020-10-01 to 2020-12-31
```

?

Reply:
> Do you mean 2020 or the whole year of 2021?

2020

> If you means 2020, it has been passed, why do we need to write a separate part of roadmap for it?

There is a roadmap for it already. I only changed the format to make all roadmaps consistent.

> I guess you want to make it as a record, but some of the features in roadmap haven't been finished, I think we need to move them out.

That's fine. It is just for history records. For example, in the future we might write more roadmaps, and we probably won't go back to delete those items that we haven't finished..